dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/entity/DependentProcessDefinition.java

```diff
@@ -60,14 +65,14 @@ public class DependentProcessDefinition {
      * get dependent cycle
      * @return CycleEnum
      */
-    public CycleEnum getDependentCycle() {
+    public CycleEnum getDependentCycle(long processDefinitionCode) {
```

Comment:
Please update the comment.
```suggestion
    public CycleEnum getDependentCycle(long upstreamProcessDefinitionCode) {
```